app/views/publishers/policy_agreements/new.html.slim

```diff
@@ -0,0 +1,19 @@
+.single-panel--wrapper
+  = render "panel_flash_messages"
+  .single-panel--content
+    .single-panel--padded-content
+      h4.single-panel--headline= t ".heading_html"
+      .text-left
+        = form_for @policy_agreement, { method: :post, url: publisher_policy_agreements_path } do |f|
+          = f.check_box(:accepted_publisher_tos, {}, 'true', 'false')
```

Comment:
Let's be sure that if there is a checkbox that the text and the checkbox are included in a `label` tag so that the user doesn't have to only click on the checkbox and can click the text too.